### ISSUE TITLE
Enhancement: Positioning of the Buttons in the save menu

### ISSUE THREAD

**Issue by AKMaily** [skunkforce/OmniView#103]
After taking a measurement, the user can click the "save" button, triggering the opening of a "save" Popup Menu. Although the menu currently possesses all the necessary functionality, its user flow is not optimal. To enhance this user flow, the buttons should be organized as described in the OmniView-Product-Vision Document under the "Toolbar/save button" section. 
The Document can be found in the "skunkforce/OmniView-Product-Vision" repository.

**Comment by DanielNowak98** [comment]
I have changed the order of the buttons in the save popup menu. Additionally, I adjusted the style to approximate the described Product Vision schema.
Removed was the condition where the Save button only appears when a device is selected.
Now, this button exists from the beginning.

Things still to be changed:

- Button with "+" to indicate additional paths needs to be added.
- Implement graying out of the Save button until devices are specified and all fields are filled.

![grafik](https://github.com/skunkforce/OmniView/assets/38864627/a8c890be-dddc-4a4a-b2e0-16e89c2c117e)
